.github/workflows/docs.yaml

```diff
@@ -44,10 +44,3 @@ jobs:
           name: docs
           path: site
           if-no-files-found: error
-      - name: Publish to GH Pages (when on main)
```

Comment:
Oh I forgot to mention in my previous comment; with this step removed, we can now take out the `permissions` section on this job (`docs`), which will make us more secure per least-privilege.